### ISSUE TITLE
Feature request: support network drives

### ISSUE THREAD

**Issue by fourg** [rjoo/teslacam-manager#1]
Rather than require a USB drive be connected it would be nice if I could point this app to my NAS CIFS shared drive where I have the TeslaCam directory.

Thanks!

**Comment by rjoo** [comment]
Good point. I hadn't considered mass storage of these recordings in that way, but it shouldn't be a problem to implement. Video playback might suffer, but I'll give a test.
